content/applications/finance/fiscal_localizations/italy.rst

```diff
@@ -122,314 +193,355 @@ Accounting.
+   of the invoice.
 
-.. warning::
-   Odoo does **not** send invoices directly to the government as they need to be signed. If we see
-   that the codice destinatario is 6 digits, then it is not sent to the PA automatically, but you
-   can download the XML, sign it with an external program and send it through the portal.
+TD28 - San Marino
+~~~~~~~~~~~~~~~~~
 
-Digital qualified signature
----------------------------
+Invoices
+********
 
-For invoices and bills intended to the **Pubblica Amministrazione (B2G)**, a **Digital Qualified
-Signature** is required for all files sent through the :abbr:`SdI (Sistema di Interscambio)`. The
-**XML** file must be certified using a certificate that is either:
+San Marino and Italy have special agreements on e-invoicing operations. As such, **invoices** follow
+the regular **reverse charge** rules. You can use the proper :guilabel:`Document Type` depending on
+the invoice type: `TD01`, `TD04`, `TD05`, `TD24`, `TD25`. Additional requirements are not enforced
+by Odoo, however, the user is requested by the **State** to:
 
-- a **smart card**;
-- a **USB token**;
-- a **Hardware Security Module (HSM)**.
+- Select a tax with the :guilabel:`Tax Exemption Kind` set to `N3.3`;
+- Use the generic :abbr:`SdI (Sistema di Interscambio)` :guilabel:`Destination Code` `2R4GT08`.
+
+The invoice is then routed by a dedicated office in San Marino to the correct business.
+
+Vendor Bills
+************
+
+When a **paper bill** is received from San Marino, any Italian company **must** submit that invoice
+to the :abbr:`AdE (Agenzia delle Entrate)` by indicating the e-invoice's :guilabel:`Document Type`
+field with the special value `TD28`.
+
+Odoo exports a transaction as `TD28` if the following conditions are met:
+
+#. It is a vendor bill;
+#. It has at least one tax on the invoice lines targets the tax grids :ref:`VJ <italy/grids>`;
+#. The **country** of the partner is **San Marino**.
+
+Public Admnistration Businesses (B2G)
+=====================================
+
+:abbr:`PA (Public Administration)` businesses are subjected to more control than private businesses
+as they handle public money coming from taxpayers. The :abbr:`EDI (Electronic Data Interchange)`
+process adds some steps to the :ref:`regular one <italy/e-invoicing-process>` as :abbr:`PA (Public
+Administration)` businesses can **accept** or **refuse** invoices.
+
+.. note::
+   :abbr:`PA (Public Administration)` businesses have a 6 digits long :guilabel:`Destination Code`
+   also called :abbr:`CUU (Codice Univoco Ufficio)` that is **mandatory**, **PEC** address cannot be
+   used in this case.
+
+.. seealso::
+   The complete list of businesses that belong to the :abbr:`PA (Public Administration)` can be
+   found here along with their :guilabel:`Destination Code`:
+   - `List of PA businesses with their Destination Code <https://www.agenziaentrate.gov.it/portale/web/guest/aree-tematiche/fatturazione-elettronica>`_
```

Comment:
I'd put the link when you say 'here' 
   The complete list of businesses that belong to the :abbr:`PA (Public Administration)` can be
   found `here   - `List of PA businesses with their Destination Code <https://www.agenziaentrate.gov.it/portale/web/guest/aree-tematiche/fatturazione-elettronica>`_
 along with their :guilabel:`Destination Code`